wandb/sdk/artifacts/artifacts_cache.py

```diff
@@ -150,7 +150,7 @@ def helper(mode: str = "w") -> Generator[IO, None, None]:
                     )
                 try:
                     os.remove(tmp_file)
-                except FileNotFoundError:
+                except (FileNotFoundError, PermissionError):
```

Comment:
is it relevant to inform the users when we aren't able to remove it? or is it will just create unnecessary confusion.